pkg/appliance/checks.go

```diff
@@ -160,10 +160,15 @@ func ShowAutoscalingWarningMessage(templateAppliance *openapi.Appliance, gateway
 	return tpl.String(), nil
 }
 
+type SkipStruct struct {
```

Comment:
```suggestion
type SkipUpgrade struct {
```
name with `Struct` stutter and is superfluous